jwst/cube_build/tests/test_configuration.py

```diff
@@ -467,3 +511,35 @@ def test_calspec3_config_nirspec_multi(tmp_cwd, nirspec_medium_coverage):
 
     assert cube_pars['1']['par1'] == ['g140m', 'g235m']
     assert cube_pars['1']['par2'] == ['f100lp', 'f170lp']
+
+
+def test_offset_file_config(tmp_cwd, miri_ifushort_short_2files, offset_file):
+    """ Test validation of the offset configuration"""
+
+    pars_input = {}
+    pars_input['channel'] = []
+    pars_input['subchannel'] = []
+    pars_input['filter'] = []
+    pars_input['grating'] = []
+    output_type = 'band'
+    weighting = 'drizzle'
+    par_filename = 'None'
+
+    pars = {
+        'channel': pars_input['channel'],
+        'subchannel': pars_input['subchannel'],
+        'grating': pars_input['grating'],
+        'filter': pars_input['filter'],
+        'weighting': weighting,
+        'output_type': output_type}
+
+    cubeinfo = cube_build.CubeData(
+        miri_ifushort_short_2files,
+        par_filename,
+        **pars)
+
+    offsets = CubeBuildStep.check_offset_file(cubeinfo.input_models)
```

Comment:
@jemorrison - this line is why you're seeing an error in the test.  Calling it this way sets `self` to `cubeinfo.input_models`.

You need to instantiate CubeBuildStep first, then you can use the instantiated object to call the `check_offset_file` function.